### DISCUSSION TITLE
Vercel blocked my github account

### DISCUSSION THREAD

**Discussion by LeanForward888** [vercel/community#4379] [Account Issue]
### Question

Hi

I try to open a vercel account using my github. For an unknown reason i got blocked on vercel at the time i tried register.  How to unlock my email so i can be able to carry on the registration procedure on vercel?

**Comment by amyegan** [comment]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)
